app/controllers/users_controller.rb

```diff
@@ -1,5 +1,12 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.all
+    render 'index'
+  end
 
-  def show; end
+  def show
+    @user = User.find(params[:id])
+    @recent_posts = user.recent_post
```

Comment:
- [ ] Please 🙏🏽 correct this line using the correct instance variable `@user` it should look like this:

```ruby
@recent_posts = @user.recent_post
```

To avoid his error 👇🏽 

![image](https://github.com/Mahamadou-Nouridine/blog-app/assets/58642949/6579bbe7-0115-4f2b-a5c5-19c10a6c0992)